scripts/scraper/venv/lib/python3.12/site-packages/_pytest/_io/saferepr.py

```
@@ -0,0 +1,130 @@
+from __future__ import annotations
+
+import pprint
+import reprlib
+
+
+def _try_repr_or_str(obj: object) -> str:
+    try:
+        return repr(obj)
+    except (KeyboardInterrupt, SystemExit):
+        raise
+    except BaseException:
```

Ruff (0.14.1)

Do not catch blind exception: `BaseException`

(BLE001)